MediaBrowser.Controller/Entities/Folder.cs

```diff
@@ -258,6 +260,7 @@ public override bool IsVisible(User user)
         /// Loads our children.  Validation will occur externally.
         /// We want this synchronous.
         /// </summary>
+        /// <returns>Returns cached children</returns>
```

Comment:
Not actually cached...
```suggestion
        /// <returns>Returns children</returns>
```